index.html

```diff
@@ -166,11 +169,21 @@ <h2 class="about_primary_text">About me</h2>
             </div>
             <div class="about_bottom">
                 <div class="text_box">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
-                <form action="" class="about_form">
-                    <input type="text" placeholder="Full name">
-                    <input type="text" placeholder="Email address">
-                    <textarea name="" id="" cols="30" rows="10">Hey I’d like to connect to...</textarea>
-                    <button class="get_in_touch"><a href="#">Get in touch</a></button>
+                <form action="https://formspree.io/f/xeqwroeb" class="about_form" method="post">
+                    <ul>
+                        <li>
+                            <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li>
+                            <input type="email" name="email" id="email" placeholder="Email address" required>
+                        </li>
+                        <li>
+                            <textarea name="textarea" id="textarea" maxlength="500" required>Hey I’d like to connect to...</textarea>
+                        </li>
```

Comment:
- [ ] [OPtional] It will be better 😃 if you remove the text you put in the textarea as when I don't write anything in the textarea and submit the required validation not work because there is a default text you put it in the textarea, so let's enhance the form by removing the text which exists in the textarea by you and this will let the validation work as expected 😊💯if you want to make this text exist you can add it as a placeholder like you did at the other inputs 😊

| Your design| 
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/228487244-a75f1304-02e0-4ad1-ba2f-2371d4d897af.png)   |